tests/strict_dbl_quotes.test/lrl.options

```diff
@@ -0,0 +1 @@
+enable_sql_stmt_caching NONE
```

Comment:
Why?

Reply:
Without this, the previously prepared statements that had double-quoted string literals will not be re-prepared, allowing it to pass incorrectly.

Reply:
makes sense.